hooks/build

```
@@ -143,12 +147,33 @@ echo
 
 # Extract python version from image name we want to build in case it was passed
 # Via IMAGE_NAME
-# This nice bash construct below extracts last field after '-' delimiter
-# So for example 'latest-3.6' will produce PYTHON_VERSION=3.6
-PYTHON_VERSION=$(echo "${LOCAL_IMAGE}" | rev | cut -d '-' -f1 | rev )
-export PYTHON_VERSION
+# This nice bash construct below extracts last field after '-python' delimiter
+# So for example 'airflow:master-python3.6' will produce LONG_PYTHON_VERSION=python3.6
+LONG_PYTHON_VERSION=$(echo "${LOCAL_IMAGE}" | rev | cut -d '-' -f1 | rev )
```

The `##` in bash does this without a pipeline https://www.tldp.org/LDP/abs/html/parameter-substitution.html

```bash
$ bash -c 'x="airflow:master-python3.6"; echo ${x##*-}'
python3.6
```

```suggestion
LONG_PYTHON_VERSION="${LOCAL_IMAGE##*-}"
```

---

Ah nice! indeed. Will change!

---

I need to learn more about all the bash string operators!